src/main/target/FF_RACEPIT/target.h

```diff
@@ -23,6 +23,9 @@
 #define USBD_PRODUCT_STRING     "RacePit"
 #define USE_TARGET_CONFIG
 
+#define USE_HARDWARE_REVISION_DETECTION
+#define HW_PIN                	PC5
```

Comment:
Having inofficial defines in here makes target maintenance harder without need for it. Please move this into `hardware_revision.c`.

Reply:
@mikeller We have multiple targets like this and all have this still in target.h. It always will go together with USE_HARDWARE_REVISION_DETECTION. Since this is really part of the configuration does it make sense to move to somewhere else? If we decide to change this we should do this for all affected targets in a separate PR?

Reply:
Yes, I know it is an encroached habit, and it is done in a number of targets, but at the bottom of it I think the approach of trying to use the minimal useful scope for everything is the correct one.
So I'd fix this in this pull request, and then fix the other targets in another one.

Reply:
Done.